dist/libphonenumber_italy.js

```diff
@@ -0,0 +1,1285 @@
+                name: "intl_number_format",
+                m: !0,
+                b: 11,
+                type: B
+            },
+            22: {
+                name: "main_country_for_code",
+                b: 8,
+                defaultValue: !1,
+                type: Boolean
+            },
+            23: {
+                name: "leading_digits",
+                b: 9,
+                type: String
+            },
+            26: {
+                name: "leading_zero_possible",
+                b: 8,
+                defaultValue: !1,
+                type: Boolean
+            }
+        }));
+        return G
+    };
+    F.ctor = F;
+    F.ctor.f = F.prototype.f;
+
+    function H() {
+        n.call(this)
+    }
+    var I;
+    m(H, n);
+    var ra = {
+        w: 1,
+        v: 5,
+        u: 10,
+        s: 20
+    };
+    H.prototype.f = function() {
+        I || (I = w(H, {
+            0: {
+                name: "PhoneNumber",
+                i: "i18n.phonenumbers.PhoneNumber"
+            },
+            1: {
+                name: "country_code",
+                required: !0,
+                b: 5,
+                type: Number
+            },
+            2: {
+                name: "national_number",
+                required: !0,
+                b: 4,
+                type: Number
+            },
+            3: {
+                name: "extension",
+                b: 9,
+                type: String
+            },
+            4: {
+                name: "italian_leading_zero",
+                b: 8,
+                type: Boolean
+            },
+            8: {
+                name: "number_of_leading_zeros",
+                b: 5,
+                defaultValue: 1,
+                type: Number
+            },
+            5: {
+                name: "raw_input",
+                b: 9,
+                type: String
+            },
+            6: {
+                name: "country_code_source",
+                b: 14,
+                defaultValue: 1,
+                type: ra
+            },
+            7: {
+                name: "preferred_domestic_carrier_code",
+                b: 9,
+                type: String
+            }
+        }));
+        return I
+    };
+    H.ctor = H;
+    H.ctor.f = H.prototype.f;
+    /*
+
+     Copyright (C) 2010 The Libphonenumber Authors
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+    */
+    var J = {
+            39: ["IT"]
```

Comment:
It's odd to me that it stores the data at `39` instead of keyed by the country, `IT`. It'd be a lot easier if the data were keyed by country.